front/src/dayview/Dayview.js

```diff
@@ -275,7 +278,7 @@ class Dayview extends Component {
 
             </div>
           </div>
-          <Link className="back" style={{ color: 'black' }} to={`/weekview/${this.state.date.toISOString().substring(0, 10)}`}>
+          <Link className={classes(css.back)} style={{ color: 'black' }} to={`/weekview/${this.state.date.toISOString().substring(0, 10)}`}>
```

Comment:
can the `color: black` be moved to scss?

Reply:
The class `.back` already contains `color: black`, I think it could be deleted safely

```
.back {
  color: $COLOR_BLACK;
}
```